Sources do not compile: CellLocation constructor error

https://github.com/j2doll/QXlsx/blob/dc31d0ff00425481d037863c191cf96e2c8d34ec/QXlsx/source/xlsxcelllocation.cpp#L20

> 3rdparty\QXlsx\QXlsx\source\xlsxcelllocation.cpp:20: error: C2679: binary '=' : no operator found which takes a right-hand operand of type 'int' (or there is no acceptable conversion)
C:\Qt\5.5\msvc2013\include\QtCore/qsharedpointer_impl.h(330): could be 'QSharedPointer<QXlsx::Cell> &QSharedPointer<QXlsx::Cell>::operator =(QSharedPointer<QXlsx::Cell> &&)'
C:\Qt\5.5\msvc2013\include\QtCore/qsharedpointer_impl.h(317): or       'QSharedPointer<QXlsx::Cell> &QSharedPointer<QXlsx::Cell>::operator =(const QSharedPointer<QXlsx::Cell> &)'
while trying to match the argument list '(QSharedPointer<QXlsx::Cell>, int)'

MSVC2013 x86 is used to compile the project.

--- a/QXlsx/source/xlsxcelllocation.cpp
+++ b/QXlsx/source/xlsxcelllocation.cpp
@@ -17,7 +17,7 @@ CellLocation::CellLocation()
     col = -1;
     row = -1;
 
-    cell = NULL;
+    cell.clear();
 }
 
 QT_END_NAMESPACE_XLSX

